src/region/__tests__/RegionCreator-test.tsx

```diff
@@ -110,6 +110,20 @@ describe('RegionCreator', () => {
             });
         });
 
+        test('should call onStart and onAbort callback when user clicks without dragging', () => {
+            const onStart = jest.fn();
```

Comment:
A common pattern in our other tests is to have a `defaults` object with the default props (which can be stubbed functions like `onAbort`). Could we make that consistent in this test class?

Reply:
Yep good catch will address.